meta/common/mojang-library-patches.json

```diff
@@ -2875,5 +2898,525 @@
       },
       "name": "org.lwjgl:lwjgl-glfw-natives-linux:3.3.2-lwjgl.1"
     }
+  },
+  {
+    "_comment": "Add linux-riscv64 support for LWJGL 3.3.3",
+    "match": [
+      "org.lwjgl:lwjgl-freetype:3.3.3"
+    ],
+    "additionalLibraries": [
+      {
+        "downloads": {
+          "artifact": {
+            "sha1": "3e3ed399be71d33bffd532039da0b7d598ba5218",
+            "size": 1194588,
+            "url": "https://github.com/Vixea/lwjgl-tmp-build/raw/main/lwjgl-freetype-natives-linux-riscv64.jar"
+          }
+        },
+        "name": "org.lwjgl:lwjgl-freetype-natives-linux-riscv64:3.3.3-lwjgl.1",
+        "rules": [
+          {
+            "action": "allow",
+            "os": {
+              "name": "linux-riscv64"
+            }
+          }
+        ]
+      }
+    ]
+  },
+  {
+    "_comment": "Add linux-riscv64 support for LWJGL 3.3.3",
+    "match": [
+      "org.lwjgl:lwjgl-glfw:3.3.3"
+    ],
+    "additionalLibraries": [
+      {
+        "downloads": {
+          "artifact": {
+            "sha1": "570be03f912e4b5aa4d4d9a837027272f9f0c15b",
+            "size": 117087,
+            "url": "https://github.com/Vixea/lwjgl-tmp-build/raw/main/lwjgl-glfw-natives-linux-riscv64.jar"
```

Comment:
Can you provide a github repo that builds these artifacts using the CI?

Reply:
That was part of the plan I just lost a bit of interest. I'll see what I can do